### PR TITLE
Add Replies feature

### PR DESCRIPTION
**Pull request by kapsteur**
Add Facebook Messenger Quick Replies feature

### CODE REVIEW

File: receiving.go [hunk 1]
```diff
@@ -55,6 +55,16 @@ type Attachment struct {
 	Payload Payload `json:"payload"`
 }
 
+// QuickReplie is a file which used in a message.
```
Comment (paked) [inline receiving.go]: Correct spelling is:

> QuickReply

File: response.go [hunk 2]
```diff
@@ -24,10 +24,16 @@ type Response struct {
 
 // Text sends a textual message.
 func (r *Response) Text(message string) error {
+	return r.TextWithReplies(message, nil)
+}
+
+// TextWithReplies sends a textual message with some replies
+func (r *Response) TextWithReplies(message string, replies *[]QuickReplie) error {
```
Comment (paked) [inline response.go]: I think it'd be neater to parse a `[]QuickReply` here, instead of a pointer.

**Comment by paked** [comment]
@kapsteur Thank you very much for the PR! It's much appreciated. I've suggested an API change and pointed out a spelling mistake - But after that we should be good to go!

File: receiving.go [hunk 3]
```diff
@@ -55,6 +55,16 @@ type Attachment struct {
 	Payload Payload `json:"payload"`
 }
 
+// QuickReply is a file which used in a message.
+type QuickReply struct {
+	// ContentType is the type of replie
```
Comment (paked) [inline receiving.go]: Sorry, just noticed you have the "replie" typo in these field descriptions too.

Comment (kapsteur) [inline receiving.go]: good catch, sorry

**Comment by paked** [comment]
Thank you @kapsteur!

**Pull request merged**